types/module/module.go

```diff
@@ -340,6 +357,16 @@ func (m *Manager) SetOrderBeginBlockers(moduleNames ...string) {
 	m.OrderBeginBlockers = moduleNames
 }
 
+// SetOrderPepareCheckStaters sets the order of set commiter calls
+func (m *Manager) SetOrderPepareCheckStaters(moduleNames ...string) {
+	m.OrderPrepareCheckStaters = moduleNames
```

Comment:
I think it is beneficial to add the assertion, like for Begin/EndBlockers, so it throws an error if a module is that implements a prepare check state / pre comitter is forgotten.
You can find an example just below.

Reply:
Good call. I also noticed this behavior was untested for the existing callbacks, so I added an assertion unit test for all of them.